src/main/scala/io/renku/jsonld/parser/JsonLDParser.scala

```diff
@@ -69,7 +69,7 @@ private class JsonLDParser() extends Parser {
     } yield JsonLD.JsonLDEntity(id, types, properties.toMap, reverse)
 
     private def extractProperties(jsonObject: JsonObject) =
-      jsonObject.toMap.view
```

Comment:
I _think_ not using `.view` is deprecated for scala 2.13 - I guess it needs to go into the compat layer as well?

Reply:
Reverted to `.view`, added needed function to compat layer for scala 2.12